.github/workflows/unittest.yml

```diff
@@ -31,6 +31,33 @@ jobs:
         name: coverage-artifacts
         path: .coverage-${{ matrix.python }}
 
+  compliance:
```

Comment:
alternatively, you could pull this into a separate file called `unittest-compliance.yml` and remove the customizations in owlbot.py for `.github/workflows/unittest.yml`. Owlbot shouldn't overwrite the changes because `unittest-compliance.yml` doesn't exist in templates.

Reply:
I'll give that a try! I was thinking mostly about having this count towards `coverage`, but I see that specifically calls out `tests/unit`, so this isn't as important.